### ISSUE TITLE
Black screen with popups

### ISSUE THREAD

**Issue by lambdadelta34** [emacs-lsp/lsp-ui#299]
- GNU Emacs 26.2
- macOS Mojave 10.14.5
- lsp-ui 20190523.1521 

lsp-ui opens strange black screen where peek popups are displayed.

![image](https://user-images.githubusercontent.com/12697325/59981332-4ef48d80-960a-11e9-8a6a-e0ba36e64f07.png)

**Comment by yyoncho** [comment]
Is it reproducible with using `emacs -q -l start-plain.el` from here - https://github.com/emacs-lsp/lsp-mode/blob/master/scripts/lsp-start-plain.el ?

**Comment by lambdadelta34** [comment]
Yes, it is. I updated that i have emacs-plus, not original gnu emacs. I'll try to reproduce with original version. Also it is reproducible with ruby.

**Comment by lambdadelta34** [comment]
Tried with original version, also reproduced.

And peek overlaps sideline.
![image](https://user-images.githubusercontent.com/12697325/59982067-5b321800-9615-11e9-9468-35788c68907e.png)

**Comment by lambdadelta34** [comment]
That's actually `lsp-ui-doc`, disabled it for now.

**Comment by SavageMessiah** [comment]
This is still happening, lsp-ui 20191125.2102, macos 10.14.6, emacs-plus 26.3. This is from spacemacs but I doubt it makes a difference. I've seen a window briefly flash up and then close when the doc overlay pops up and emacs is not fullscreen. When its fullscreen another black fullscreen window opens up and somehow steals the menubar - if I put the cursor at the top of the screen in the actual useful emacs window the menubar does not come down, but if I do that on the black screen it does what I would expect. Weird. I think the first time this happened, emacs prompted me to grant it accessibility access, if that means anything.

I'm going to disable the doc overlay for now as well, but it would be nice to get this fixed, the overlay is useful.

**Comment by tbtommyb** [comment]
I am experiencing the exact same behaviour as @SavageMessiah with lsp-ui-20200115.1352, macOS 10.14.2 and GNU Emacs 26.2. Happy to do some debugging if anyone can give me a pointer.

**Comment by kangxiaoning** [comment]
I am experiencing the exact same behaviour as @SavageMessiah with `lsp-ui-20200115.1352`, `macOS 10.15.3` and `GNU Emacs 26.3`.

**Comment by seabadger** [comment]
>When its fullscreen another black fullscreen window opens up and somehow steals the
> menubar - if I put the cursor at the top of the screen in the actual useful emacs window the
> menubar does not come down, but if I do that on the black screen it does what I would 
> expect.
I'm experiencing the same. MacOS 10.15.5, Emacs for MacOS X (GNU Emacs 26.3 (build 1, x86_64-apple-darwin18.2.0, NS appkit-1671.20 Version 10.14.3 (Build 18D109)) of 2019-09-02)

**Comment by petsgre** [comment]
I have the same situation.
when i fullscreen the emacs. It will open a new window
emacs 26.3 
macOS High Sierra 10.13.6

**Comment by ilkka** [comment]
FWIW this happens with spacemacs + eshell + company as well, so it's not just an LSP thing

**Comment by siraben** [comment]
I have this same issue with `lsp-ui` on

Emacs 27.1
macOS Catalina 10.15.6

**Comment by simpsonw** [comment]
Same behavior using Spacemacs develop branch on macOS Big Sur (11.4) with Emacs 27.2.
